Example/GiniVision/ComponentAPIAnalysisViewController.swift

```diff
@@ -92,7 +92,7 @@ class ComponentAPIAnalysisViewController: UIViewController {
         hideErrorButton()
         
         // Retry analysis of the document.
-        AnalysisManager.sharedManager.analyzeDocument(withImageData: imageData, cancelationToken: CancelationToken(), completion: nil)
+        AnalysisManager.sharedManager.analyzeDocument(withData: document.data, cancelationToken: CancelationToken(), completion: nil)
```

Comment:
I guess `sharedManager` is not really Swift-3-like. We might consider renaming it to `shared`. But in another PR.